tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+                    if url.startswith("data:image"):
+                        # Base64 image
+                        media_type, data = url.split(",", 1)
+                        content.append(
+                            {
+                                "type": "image",
+                                "source": {
+                                    "type": "base64",
+                                    "media_type": "image/png",
+                                    "data": data,
+                                },
+                            }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)